code/datums/mutations/_mutations.dm

```diff
@@ -197,18 +196,31 @@
 			apply_overlay(mutation.layer_used)
 
 /**
- * Called when a chromosome is applied so we can properly update some stats
- * without having to remove and reapply the mutation from someone
- *
- * Returns `null` if no modification was done, and
- * returns an instance of a power if modification was complete
+ * Called after on_aquiring, or when a chromosome is applied so we can
+ * properly update some stats without having to remove and reapply the mutation from someone
+ * returns the instance of 'power_path' for children calls to use without calling locate() again.
  */
-/datum/mutation/human/proc/modify()
+/datum/mutation/human/proc/setup(obj/item/chromosome/chromosome)
 	if(modified || !power_path || QDELETED(owner))
 		return
+
+	if(chromosome)
+		if(stabilizer_coeff != MUTATION_COEFFICIENT_UNMODIFIABLE)
+			stabilizer_coeff = chromosome.stabilizer_coeff
+		if(synchronizer_coeff != MUTATION_COEFFICIENT_UNMODIFIABLE)
+			synchronizer_coeff = chromosome.synchronizer_coeff
+		if(power_coeff != MUTATION_COEFFICIENT_UNMODIFIABLE)
+			power_coeff = chromosome.power_coeff
+		if(energy_coeff != MUTATION_COEFFICIENT_UNMODIFIABLE)
+			energy_coeff = chromosome.energy_coeff
+		can_chromosome = CHROMOSOME_USED
+		chromosome_name = chromosome.name
+		modified = TRUE
+		qdel(chromosome)
```

Comment:
ignoring context of this pr as cleanup, this strikes me as kind of an unsafe pattern, we're taking an input which we decompose and then delete, but any overrides of this proc ALSO have htat input, and they just have to know they're notallowed ot ever touch it at risk of harddels or weird behavior. it seems like the wrong solution here, what's the motivation?